bmxx80/bmxx80.go

```diff
@@ -20,6 +20,9 @@ import (
 	"periph.io/x/conn/v3/spi"
 )
 
+// I2CAddr i2c default address.
```

Comment:
Can you make a separate PR for this and the file below? They are unrelated to firmata.

Reply:
Didn't mean for that to make it in, will move to another PR.